core/migrations/0018_auto_20180731_1400.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.0.1 on 2018-07-31 14:00
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0017_auto_20180726_1244'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='OfficeFloorSection',
+            fields=[
+                ('id', models.AutoField(
+                    auto_created=True,
+                    primary_key=True,
+                    serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=100)),
+                ('floor', models.ForeignKey(
+                    on_delete=django.db.models.deletion.PROTECT,
+                    to='core.OfficeFloor')),
+            ],
+            options={
+                'verbose_name': 'Floor Section',
```

Comment:
The change of the verbose_name requires generating a new migration file for the name change to reflect. I recommend, you delete this migration file and generate a new one.

Reply:
Also, I have noticed you are force pushing your changes...could you just create new commits instead, there's no harm in having many commits.